solutions/security-enforced/variables.tf

```diff
@@ -181,7 +181,7 @@ variable "existing_event_notifications_instance_crn" {
   default     = null
 }
 
-variable "skip_event_notifications_iam_authorization_policy" {
+variable "skip_secrets_manager_event_notifications_iam_authorization_policy" {
```

Comment:
getting very long, maybe shorten `iam_authorization` -> `iam_auth` in the name? same for the other related inputs

Reply:
in both DA variations of course